.github/stale.yml

```diff
@@ -0,0 +1,57 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
```

Comment:
Seems like quite a long time tbh - 30 days might suffice unless folk are off backpacking the world between PRs ;)

Oh - now I see the PR specific value for this variable at the end. Might be worth adding a comment that we're only using this value for issues (and I guess that 60 days now seems reasonable).